commands/release.js

```diff
@@ -133,7 +134,8 @@ function release(context) {
     cli.log('creating remote slug...');
     cli.log('remote process types:', modifiedProc);
     var slugInfo = app.slugs().create({
-      process_types: modifiedProc
+      process_types: modifiedProc,
+      buildpack_provided_description: appJSON.image || 'docker (unknown)'
```

Comment:
At the very least I'd like to consider changing this to:

"heroku-docker (appJSON.image || unknown)"